Crash on server

Minecraft 1.20.1
java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'edenring'!
       at net.fabricmc.loader.impl.FabricLoaderImpl.lambda$invokeEntrypoints$2(FabricLoaderImpl.java:388)
       at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33)
       at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:386)
       at net.fabricmc.loader.impl.game.minecraft.Hooks.startServer(Hooks.java:63)
       at net.minecraft.server.Main.main(Main.java:112)
       at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
       at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
       at net.fabricmc.loader.impl.launch.knot.KnotServer.main(KnotServer.java:23)
       at net.fabricmc.loader.impl.launch.server.FabricServerLauncher.main(FabricServerLauncher.java:69)
Caused by: java.lang.NoSuchFieldError: CLIENT_CONFIG
       at paulevs.edenring.config.Configs.<clinit>(Configs.java:20)
       at paulevs.edenring.registries.EdenItems.<clinit>(EdenItems.java:13)
       at paulevs.edenring.registries.EdenBlocks.<clinit>(EdenBlocks.java:48)
       at paulevs.edenring.EdenRing.onInitialize(EdenRing.java:49)
       at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:384)
       ... 6 more

--- a/gradle.properties
+++ b/gradle.properties
@@ -20,7 +20,7 @@ loader_version=0.15.6
 fabric_version=0.91.0+1.20.1
 
 # Mod Properties
-mod_version=0.6.8
+mod_version=0.6.9
 maven_group=paulevs.edenring
 archives_base_name=eden-ring
 

--- a/src/main/java/paulevs/edenring/EdenRing.java
+++ b/src/main/java/paulevs/edenring/EdenRing.java
@@ -24,14 +24,7 @@ import net.minecraft.world.level.storage.loot.providers.number.ConstantValue;
 import org.betterx.worlds.together.world.WorldConfig;
 import paulevs.edenring.config.Configs;
 import paulevs.edenring.paintings.EdenPaintings;
-import paulevs.edenring.registries.EdenBiomes;
-import paulevs.edenring.registries.EdenBlockEntities;
-import paulevs.edenring.registries.EdenBlocks;
-import paulevs.edenring.registries.EdenEntities;
-import paulevs.edenring.registries.EdenFeatures;
-import paulevs.edenring.registries.EdenItems;
-import paulevs.edenring.registries.EdenRecipes;
-import paulevs.edenring.registries.EdenSounds;
+import paulevs.edenring.registries.*;
 import paulevs.edenring.world.EdenPortal;
 import paulevs.edenring.world.generator.EdenBiomeSource;
 import paulevs.edenring.world.generator.GeneratorOptions;
@@ -67,6 +60,7 @@ public void onInitialize() {
   EdenItems.init();
   EdenFeatures.register();
 //  EdenRecipes.register(); Use data generation
+  EdenParticles.ensureStaticallyLoadedServerside();
   Configs.saveConfigs();
   
   Registry.register(BuiltInRegistries.BIOME_SOURCE, makeID("biome_source"), EdenBiomeSource.CODEC);

--- a/src/main/java/paulevs/edenring/blocks/VolvoxBlock.java
+++ b/src/main/java/paulevs/edenring/blocks/VolvoxBlock.java
@@ -32,7 +32,6 @@ import java.util.Optional;
 public class VolvoxBlock extends SlimeBlock implements RenderLayerProvider, BlockModelProvider, CustomColorProvider {
 	private static final OpenSimplexNoise NOISE = new OpenSimplexNoise("volvox".hashCode());
 	
-	@Environment(EnvType.CLIENT)
 	private static final int[] COLORS;
 	
 	public VolvoxBlock() {

--- a/src/main/java/paulevs/edenring/config/Configs.java
+++ b/src/main/java/paulevs/edenring/config/Configs.java
@@ -16,7 +16,6 @@ public class Configs {
     public static final PathConfig ITEMS = new PathConfig(EdenRing.MOD_ID, "items");
     public static final PathConfig RECIPES = new PathConfig(EdenRing.MOD_ID, "recipes");
 
-    @Environment(EnvType.CLIENT)
     public static final EdenClientConfig CLIENT_CONFIG = new EdenClientConfig();
 
     public static void saveConfigs() {

--- a/src/main/java/paulevs/edenring/registries/EdenParticles.java
+++ b/src/main/java/paulevs/edenring/registries/EdenParticles.java
@@ -14,4 +14,7 @@ public class EdenParticles {
 	private static SimpleParticleType register(String name) {
 		return Registry.register(BuiltInRegistries.PARTICLE_TYPE, EdenRing.makeID(name), FabricParticleTypes.simple());
 	}
+
+	public static void ensureStaticallyLoadedServerside() {
+	}
 }

